packages/console/src/pages/ApplicationDetails/SamlApplicationDetailsContent/Settings.tsx

```diff
@@ -280,6 +314,77 @@ function Settings({ data, mutateApplication, isDeleted }: Props) {
               }}
             />
           </FormField>
+          <FormField title="application_details.saml_idp_name_id_format.title">
+            <Controller
+              name="nameIdFormat"
+              control={control}
+              render={({ field: { onChange, value } }) => (
+                <Select
+                  options={Object.values(NameIdFormat).map((format) => ({
+                    value: format,
+                    // eslint-disable-next-line no-restricted-syntax
+                    title: (
+                      <span>
+                        {t(nameIdFormatToOptionMap[format])}
+                        <span className={styles.nameIdFormatDescription}>
+                          ({t(nameIdFormatToOptionDescriptionMap[format])})
+                        </span>
+                      </span>
+                    ) as React.ReactNode,
```

Comment:
Is this `as React.ReactNode` necessary? Seems useless in this case, as the `<span>` IS react node